shared_model/interfaces/permissions.hpp

```diff
@@ -60,6 +60,7 @@ namespace shared_model {
         kGetBlocks,
         kAddDomainAssetQty,
         kSubtractDomainAssetQty,
+
```

Comment:
An empty line added. That looks very suspicious.

Reply:
Added for consistency with `enum class Grantable` values below.